README.md

```diff
@@ -55,230 +57,82 @@ Log4j
-is no evaluation with provided id, the default value is returned.
-
-Use the appropriate method to fetch the desired Evaluation of a certain type.
-
-### Bool variation
-
-```
-boolean result = cfClient.boolVariation("sample_boolean_flag", target, false);  
-```
-
-### Number variation
-
-```
-double result = cfClient.numberVariation("sample_number_flag", target, 0);  
-```
-
-### String variation
-
-```
-boolean result = cfClient.stringVariation("sample_string_flag", target, "");  
-```
+This quickstart assumes you have followed the instructions to [setup a Feature Flag project and have created a flag called `harnessappdemodarkmode` and created a server API Key](https://ngdocs.harness.io/article/1j7pdkqh7j-create-a-feature-flag#step_1_create_a_project).
 
-## Using feature flags metrics
+### Install the FF SDK Dependency
 
-Metrics API endpoint can be changed like this:
-
-```
-Config.builder()
-              .eventUrl("METRICS_API_EVENTS_URL")
-              .build();
-```
-
-Otherwise, the default metrics endpoint URL will be used.
-
-## Listen on events
-
-```
-client.on(Event.READY, result -> log.info("READY"));
-
-client.on(Event.CHANGED, result -> log.info("Flag changed {}", result));
-```
+The first step is to install the FF SDK as a dependency in your application using your application's dependency manager. You can use Maven, Gradle, SBT, etc. for your application.
 
-events will work even when streamEnabled is off.
+Refer to the [Harness Feature Flag Java Server SDK](https://mvnrepository.com/artifact/io.harness/ff-java-server-sdk) to identify the latest version for your build automation tool.
 
-## Connector
+This section lists dependencies for Maven and Gradle and uses the 1.1.4 version as an example:
 
-This is a new feature that allows you to create or use other connectors.
-Connector is just a proxy to your data. Currently supported connectors:
-* Harness
-* Local (used only in development)
+#### Maven
 
+Add the following Maven dependency in your project's pom.xml file:
+```pom
+<dependency>
+    <groupId>io.harness</groupId>
+    <artifactId>ff-java-server-sdk</artifactId>
+    <version>1.1.4</version>
+</dependency>
 ```
-LocalConnector connector = new LocalConnector("./local");
-client = new CfClient(connector);
-```
-
-## Storage
 
-For offline support and asynchronous startup of SDK use storage interface.
-When SDK is used without waitForInitialization method then it starts in async mode.
-So all flags are loaded from last saved configurations and it will use that values.
-If there is no flag on storage then it will be evaluated from defaultValue argument.
+#### Gradle
 
 ```
-final FileMapStore fileStore = new FileMapStore("Non-Freemium");
-LocalConnector connector = new LocalConnector("./local");
-client = new CfClient(connector, Config.builder().store(fileStore).build());
+implementation group: 'io.harness', name: 'ff-java-server-sdk', version: '1.1.4'
```

Comment:
I never test it using gradle but I remember there were some deps with gradle:
```
allprojects {
    repositories {
        ...
        maven { url 'https://jitpack.io/' }
    }
}
``` maybe @milos85vasic can confirm because he already use gradle on some Android app

Reply:
I will remove Gradle steps until we can test this. When this gets resolved, we'll have to update the main docs as well.
https://ngdocs.harness.io/article/i7et9ebkst-integrate-feature-flag-with-java-sdk#gradle